reference-architecture/vmware-ansible/vars/main.yaml

```diff
@@ -0,0 +1,53 @@
+---
+# These will be the variables that can be overloaded via the wrapper script
+vcenter_host: '10.19.114.221'
+vcenter_username: 'administrator@vsphere.local'
+vcenter_password: 'P@ssw0rd'
```

Comment:
With these values not working for the general use case, I'd almost say that it isn't worth defining them, rather just documenting that they would need to be set using the wrapper script, or through inventory variables.